cfpb/consumerfinance.gov
# Update DS@3.5.0, eslint/js@9.15.0, cypress@13.16.0, eslint@9.15.0, eslint-plugin-jsdoc@50.6.0, immutable@5.0.3, prettier@3.4.1, snyk@1.1294.2

Brings in DS v3.5.0 changes https://github.com/cfpb/design-system/blob/main/packages/cfpb-design-system/CHANGELOG.md#350---november-26-2024

Fixes https://github.com/cfpb/consumerfinance.gov/security/dependabot/112

## Changes

- Update DS@3.5.0, eslint/js@9.15.0, cypress@13.16.0, eslint@9.15.0, eslint-plugin-jsdoc@50.6.0, immutable@5.0.3, prettier@3.4.1, snyk@1.1294.2
- Autofix linter issues.


## How to test this PR

1. PR checks should pass. 
2. `yarn build` and edit a hero such as on the homepage to have a very short heading and subheading and see that the text shouldn't be center-aligned.
